README.md

```diff
@@ -4,9 +4,146 @@ CartoDB Spatial Analysis extension for PostgreSQL.
 
 ## Code organization
 
-* *pg* contains the PostgreSQL extension source code
-* *python* Python module
+* *doc* documentation
+* *src* source code
+* - *src/pg* contains the PostgreSQL extension source code
+* - *src/py* Python module source code
+* *release* reseleased versions
+* *env* base directory for Python virtual environments
 
 ## Requirements
 
-* pip
+* pip, virtualenv, PostgreSQL
+* python-scipy system package (see src/py/README.md)
+
+# Working Process
+
+We use the default `develop` branch as the basis for development.
+This branch and `master` are maintained by the *Release Manager*.
```

Comment:
I think the developers should branch off develop and merge back to it, but al merges between develop and master and direct commits to develop should be handled by the RM (release manager)